### PR TITLE
Obscure password

### PR DESCRIPTION
**Pull request by bkorty**
This MR overwrites the password with ***** when the postgres connection DSN is written to log files. It now also prints out the safe DSN when the application fails to connect at startup.

### CODE REVIEW

File: sprockets_postgres.py [hunk 1]
```diff
@@ -475,13 +477,25 @@ async def _postgres_connect(self) -> bool:
...
         self._postgres_connected.set()
         LOGGER.debug('Connected to Postgres')
         return True
 
+    @staticmethod
+    def _obscure_url_password(url):
+        """Generate log safe url with password obscured."""
+        parsed = parse.urlparse(url)
+        if parsed.password:
+            netloc = '{}:*****@{}:{}'.format(parsed.username,
```
Comment (gmr) [inline sprockets_postgres.py:493]: super tiny nit, but does it make sense to make the masked password the same length as the password? I'm not sure of a best practice here, but the thought is at least you know the password length matches when it's masked. Then again that may be too much information disclosure or not worth the effort.

Comment (andrewrabert) [reply sprockets_postgres.py:493]: I'm in favor of simplicity here - using a constant string. Arguably safe too.

Comment (bkorty) [reply sprockets_postgres.py:493]: I'm inclined to stick with the constant string for security reasons. I was even wondering if I should partially mask the user name. But I can change if you want.

Comment (dave-shawley) [reply sprockets_postgres.py:493]: FWIW, I'm in favor of the constant string as well.  Simpler and doesn't expose any information whatsoever.

**Pull request merged**